File: 250319/정수 사이의 배수/multiplication-between-numbers.cpp
#include <iostream>
using namespace std;

int main()
{
    ios::sync_with_stdio(false);
    cin.tie(0);
    cout.tie(0);

    int a = 0, b = 0, sum = 0, count = 0;
    double avg = 0;

    cin >> a >> b;
    for(int i = a; i <= b; i++) {
        if(i % 5 == 0 || i % 7 == 0) {
            sum += i;
            count++;
        }
    }
    cout << fixed;
    cout.precision(1);
    avg = (double)sum / count;
    cout << sum << " " << avg;
    return 0;
}

File: 250319/정수 사이의 배수/multiplication-between-numbers.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int a = sc.nextInt();
        int b = sc.nextInt();
        int sum = 0;
        int count = 0;
        double avg = 0;
        for(int i = a; i <= b; i++) {
            if(i % 5 == 0 || i % 7 == 0) {
                sum += i;
                count++;
            }
        }
        avg = (double)sum / count;
        System.out.printf("%d %.1f", sum, avg);
        sc.close();
    }
}